app.py

```diff
@@ -14,7 +14,9 @@
 st.image(pil_image, caption="Input image", use_column_width=True)
 preds = components.make_prediction(model, pil_image)
 
-components.display_prediction(preds)
+label = components.get_prediction(preds)
```

Comment:
I expect there's going to be a merge conflict once we merge this PR or #54, looks like we both ran into the same problem needing to access that variable.